### ISSUE TITLE
Is it working on LXC

### ISSUE THREAD

**Issue by azlux** [aristocratos/bashtop#7]
Hi,
I trying your project into a LXC container.
I have this error :
```
New instance of bashtop version: 0.6.11 Pid: 25782
10:49:43 PM ERROR: On line 180
./bashtop: line 1292: ( 100*(60649195923-60649210126-+) ) / (60649195923-60649210126) : syntax error: operand expected (error token is ") ) / (60649195923-60649210126) ")
10:49:44 PM ERROR: On line 385
```
Try with : GNU bash, version 5.0.3(1)-release (x86_64-pc-linux-gnu)

Here is tracing.log file  (it's quite huge):
[tracing.log](https://github.com/aristocratos/bashtop/files/4458641/tracing.log)

**Comment by aristocratos** [comment]
There seems to be an issue with /proc/stat reporting bad values for cpu idle on certain cores. Could be related to [https://github.com/lxc/lxcfs/issues/311](https://github.com/lxc/lxcfs/issues/311)
I could add an error check, but that wouldn't fix the problem of /proc/stat reporting wrong cpu usage.

**Comment by aristocratos** [comment]
fix in [5cda57c](https://github.com/aristocratos/bashtop/commit/5cda57c8844bb555b7cea5915bd2069199e5d1fc)
But will just skip reporting cpu usage for threads with idle cpu errors. Root cause to the problem is with LXC.
